onnxruntime/core/providers/openvino/backends/vadm_backend.cc

```diff
@@ -260,4 +260,4 @@ void VADMBackend::Infer(Ort::CustomOpApi& ort, OrtKernelContext* context) {
 }
 
 }  // namespace openvino_ep
-}  // namespace onnxruntime
\ No newline at end of file
+}  // namespace onnxruntime
```

Comment:
seems like only formatting change to this file? is it really necessary?

Reply:
removed the file from commit